src/components/accessories/admin/types/components/agetypes/ageTypesForm/types.ts

```diff
@@ -0,0 +1,22 @@
+import { FormikProps } from "formik";
+import { TFunction } from "react-i18next";
+import { AgeTypeDTO } from "../../../../../../../generated";
+import { TFields } from "../../../../../../../libraries/formDataHandling/types";
+
+export interface IAgeTypesFormProps {
+  rows: TFields<AgeTypeFormFieldName>[];
+  onSubmit: (ageType: AgeTypeDTO[]) => void;
+  submitButtonLabel: string;
+  resetButtonLabel: string;
+  isLoading: boolean;
+}
+
+export interface IAgeTypeFormProps {
+  t: TFunction;
+  isValid: (fieldName: string, index: number) => boolean;
+  getErrorText: (fieldName: string, index: number) => string;
+  formik: FormikProps<{ ageTypes: Record<string, any>[] }>;
+  index: number;
+}
```

Comment:
```suggestion
export interface IAgeTypeFieldProps {
  value: ...;
  onChange: (fieldName: string, index: number, value: ...) => boolean;
  isValid: (fieldName: string, index: number) => boolean;
  getErrorText: (fieldName: string, index: number) => string;
}
```

Reply:
Which type is this "..."?

Reply:
Could be number | string (type of ageType.min or ageType.description)